cdc/changefeed.go

```diff
@@ -886,7 +886,7 @@ func (c *changeFeed) calcResolvedTs(ctx context.Context) error {
 		c.ddlJobHistory = c.ddlJobHistory[1:]
 	}
 	if len(c.ddlJobHistory) > 0 && minResolvedTs >= c.ddlJobHistory[0].BinlogInfo.FinishedTS {
-		minResolvedTs = c.ddlJobHistory[0].BinlogInfo.FinishedTS
+		minResolvedTs = c.ddlJobHistory[0].BinlogInfo.FinishedTS - 1
```

Comment:
Could you add some comments about `=1`?

Reply:
This is a hotfix, not this PR should fix, ref: https://github.com/pingcap/ticdc/pull/1507

Reply:
I have reverted it